packages/array-index/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "just-index",
+  "version": "1.0.0",
+  "description": "returns an object keyed from array of objects",
```

Comment:
could be clearer maybe? "return an object from an array, keyed by the value at the given id".